python/release/wheel/README.md

```diff
@@ -0,0 +1,16 @@
+Description
+------------------------------
+This directory is used to build released wheels according to PEP513 and upload
+them to pypi.
+
+Usage
+------------------------------
+    ./protobuf_optimized_pip.sh PROTOBUF_VERSION PYPI_USERNAME PYPI_PASSWORD
```

Comment:
Can you add an example here? It's not clear what PROTOBUF_VERSION is. Is it "3.3.0", "v3.3.0", "protobuf-3.3.0", "protobuf-v3.3.0" or something else?